src/acars_router_servers/tcp/outputs/tcp_serve_server.rs

```diff
@@ -120,20 +119,25 @@ impl TCPServeServer {
     }
 }
 
-async fn handle_message(state: Arc<Mutex<Shared>>, mut channel: Receiver<serde_json::Value>) {
+async fn handle_message(state: Arc<Mutex<Shared>>, mut channel: Receiver<Value>) {
     loop {
-        let message = channel.recv().await;
-
-        if let Some(message) = message {
-            let message_out = message["out_json"].clone();
-            let message_as_string = format!("{}\n", message_out);
-            state.lock().await.broadcast(&message_as_string).await;
+        let message: Option<Value> = channel.recv().await;
+
+        if let Some(received_message) = message {
+            let message_out: Value = received_message["out_json"].clone();
+            let message_as_string: Result<String, serde_json::Error> = serde_json::to_string(&message_out);
```

Comment:
Can probably simplify this even further by making it:

`let message_as_string: Result<String, serde_json::Error> = serde_json::to_string(&received_message["out_json"]);`

See how it performs.